test/vni-manager-mocha.js

```diff
@@ -0,0 +1,377 @@
+/**
+ * File: vni-manager-mocha.js
+ * Unit tests for the vni APIs defined in src/vni-manager.js
+ */
+
+var chai = require('chai');
+var expect = chai.expect;
+var should = chai.should();
+
+var componentFactory = require('../src/noflo-component-factory');
```

Comment:
Let's rename noflo-component-factory --> noflo-node-factory
and componentFactory --> nodeFactory

But let's do it in a separate PR